### PR TITLE
Angular + Bootstrap Template Added

### PR DESCRIPTION
**Pull request by PMArtz92**
i have used bower to manage the front end dependencies.

### CODE REVIEW

**Comment by mariobehling** [comment]
@PMArtz92 Thank you. Please resolve the conflicts in the Readme.md, so we can merge it. Cheers.

**Comment by daminisatya** [comment]
@PMArtz92 gave a review comment and also please add screenshots. Thank you.

**Comment by daminisatya** [comment]
@PMArtz92 Few things which i would like to mention in common. 

* Make sure the file endings are proper
* Try to include the minified versions of library files. 
* Add screenshots
* Have check on the formatting for a common indentation. 

Please resolve these and it's good to go from my side.

**Comment by PMArtz92** [comment]
please check now i have resolved the conflict

**Comment by mariobehling** [comment]
@PMArtz92 Still showing: This branch has conflicts that must be resolved

**Comment by sudheesh001** [comment]
6 commits to push a boilerplate into the app ? Please squash the commits into one and this should be good to merge.

File: src/app/core/scss/bootstrap/_card.scss [hunk 1]
```diff
@@ -0,0 +1,294 @@
...
+}
+
+// .card-actions {
+//   padding: $card-spacer-y $card-spacer-x;
+
+//   .card-link + .card-link {
+//     margin-left: $card-spacer-x;
+//   }
+// }
+
```
Comment (sudheesh001) [inline src/app/core/scss/bootstrap/_card.scss]: Any reason why this part of the code is commented out ?

**Pull request merged**